### DISCUSSION TITLE
❓ Questions / Help / Support

### DISCUSSION THREAD

**Discussion by sunnnnnnnny** [snakers4/silero-vad#562] [Q&A] [answered]
## ❓ Questions and Help

If i want to vad and keep little silence with chunk_wav's begining and ending, how do it?

**Comment by snakers4** [comment, marked as answer]
https://github.com/snakers4/silero-vad/blob/e531cd3462189f275e0a231f0cefb21816147ed2/src/silero_vad/utils_vad.py#L187-L251

try fiddling with `min_silence_duration_ms`  and `speech_pad_ms`.
